Plugins/NativeCamera/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+set(SOURCES 
+    "Include/Babylon/Plugins/NativeCamera.h"
+    "Source/NativeCamera.cpp"
+    "Source/NativeCamera.h"
+    "Source/NativeVideo.cpp"
+    "Source/NativeVideo.h")
+
+if(APPLE)
+    set(SOURCES ${SOURCES} "Source/NativeCamera${BABYLON_NATIVE_PLATFORM}.mm")
```

Comment:
I think this is captured elsewhere in feedback, but I believe the naming and directory structure we've been settling on is to have one directory per platform, have the file names all be the same, and put platform agnostic code in a `shared` folder.

Reply:
We have two structures. I don't think we have to settle to exact one format.

Reply:
Historically, I think we've done the `Shared` folder thing mainly with headers, handling platforms in `Source` folders more akin to how it's done here. We might not have been as consistent about that as I'm thinking, though.